misc/create_burnman_readable_perplex_table.py

```diff
@@ -0,0 +1,86 @@
+from subprocess import Popen, PIPE, STDOUT
+import argparse
+
+'''  
+This python file uses PerpleX's werami software 
```

Comment:
so this is a command line utility, right? Would it make sense to have the functionality below be a python function in burnman/perplex.py and call it from this file? Or is it not useful to call this programmatically?